opba-db/src/main/resources/migration/migrations/0002-mock-data.xml

```diff
@@ -148,4 +148,74 @@
             </where>
         </update>
     </changeSet>
+
+    <changeSet author="iiuex@adorsys.com.ua" id="2020-08-21-1">
```

Comment:
This changeSet should be triggered conditionally, and should not be in `0002-mock-data.xml` it is real banks data, not mock-banks

Reply:
Added context field to the changeSet